### ISSUE TITLE
Feature Request: Don't load fzf-lua when ecolog start

### ISSUE THREAD

**Issue by vunhatchuong** [philosofonusus/ecolog.nvim#6]
Currently, if you enable `fzf-lua` integration, if you load `ecolog` it will also load `fzf-lua` because of the check: https://github.com/philosofonusus/ecolog.nvim/blob/f6e608dd60d16f1721b64e0314fe649283495ff2/lua/ecolog/integrations/fzf.lua#L1

Is it possible to only load it when the user runs `:EcologFzf`.

**Comment by philosofonusus** [comment]
Yes, that was quite simple to accomplish, this update (https://github.com/philosofonusus/ecolog.nvim/commit/c63242c205f516fac2e9035f4bb3a73a27922cd7) enables lazy-loading for fzf integration by default

**Comment by vunhatchuong** [comment]
Works perfectly, thanks!
